index.yaml

```diff
@@ -4158,6 +4158,37 @@ entries:
     - assets/longhorn-crd/longhorn-crd-1.0.200.tgz
     version: 1.0.200
   neuvector:
+  - annotations:
+      catalog.cattle.io/auto-install: neuvector-crd=match
+      catalog.cattle.io/certified: rancher
+      catalog.cattle.io/display-name: NeuVector
+      catalog.cattle.io/kube-version: '>=1.18.0-0 < 1.29.0-0'
```

Comment:
< 1.29 k8s version, is this right?

Reply:
It is. The chart supports k8s versions up to 1.28.